docs/release-notes.rst

```diff
@@ -1,3 +1,289 @@
+
+If a node (for example a node with IP 1.2.3.4) is deleted from APIC, and there is also another node 1.2.3.40 of which the original IP is a substring, it may be possible that the dynamic end point attach detach feature is not able to delete 1.2.3.4 from BIG-IP. Note: The pool members will get deleted as expected. 
+
+**Workaround:** Login to the BIG-IP UI and delete the problematic node.
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/244
+
+------
+
+**Dynamic EP discovery does not work if a duplicate IP already exists on a different partition**
+
+If an APIC Tenant|App|EPG mapped to a BIG-IP pool has an endpoint with an IP address which already exists on the BIG-IP but in a different partition, then the APIC endpoint will not get added to BIG-IP pool. Also any successive configurations and endpoints also will not be discovered/deleted from this BIG-IP pool. 
+
+Workaround: Remove the duplicate IPs from the endpoint list on APIC and retry a manual sync of Endpoints from L4-L7 App Services --> Application Inventory --> Sync EPs icon. 
+
+Note: Similar issues might be seen with other erroneous configurations such as unsupported IPv4 formats like 1.2.3.4/24 instead of 1.2.3.4
+
+AS3 Defect: https://github.com/F5Networks/f5-appsvcs-extension/issues/287
+
+------
+
+**Pool members deleted or added directly to BIG-IP don't get updated automatically after clicking "Sync EPs"**
+
+1. If BIG-IP pool members are automatically added by the **Dynamic endpoint discovery** feature, but then a few endpoints are deleted directly from the BIG-IP (i.e. out of band); these endpoints do not get created again when clicking **L4-L7 App Services --> Application Inventory --> Sync EPs** for that application. 
+
+2. Similarly, when a few pool members are added directly to the BIG-IP (i.e. out of band), these extra members are not deleted after clicking **L4-L7 App Services --> Application Inventory --> Sync EPs**
+
+**Workaround:** Manually add/delete the pool members from BIG-IP. 
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/293
+
+------
+
+**Pool members are not synced on AS3 service discovery REST API endpoint for HA devices**
+
+AS3 Service Discovery REST API endpoint on both HA devices should display the same pool member list for the specified pool path (For. ex. https://BIG-IP/mgmt/shared/service-discovery/task/~Partition~Application~Pool). But AS3 service discovery fails to perform this sync between the HA devices.
+
+**Workaround:** None.
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/385
+
+------
+
+**FAST input fields do not display any validation errors until the field is touched**
+
+Due to Angular Forms behaviour, for FAST input fields that have any validations, display a validation error message only when that field is clicked/touched explicitly. There is no indication given to the user for the validations that are applied to these fields. 
```

Comment:
Due to Angular Forms behavior, FAST input fields that have any validation display a validation error message only...

...to the user for the validation that is applied to these fields.